config/core/test-configs.yaml

```diff
@@ -756,6 +764,7 @@ device_types:
     mach: x86
     arch: x86_64
     boot_method: depthcharge
+    context: {extra_kernel_args: cros_debug cros_secure root=/dev/mmcblk0p3}
```

Comment:
@gctucker I am not sure if we can keep this context here..

Using [artifact and bmeta json files](https://people.collabora.com/~alex.pereira/chromeos/chromeos-images/) availables on my page it is possible to build LAVA job template using `kci_test` and run it in a lab.

Last test results:
https://lava.collabora.co.uk/scheduler/job/5763022

Reply:
Yes, these parameters should be moved to the chromeos-tast test plan definition. Maybe it could be added to the `context` Jinja2 dictionary variable in `chromeos-tast-template.jinja2`? I think it's fine to hard-code it in this template since it's going to be the same for all Chrome OS tests.

Reply:
To check your changes, you could also generate and submit a baseline test for the same device and same kernel meta-data etc.

Reply:
> Yes, these parameters should be moved to the chromeos-tast test plan definition. Maybe it could be added to the `context` Jinja2 dictionary variable in `chromeos-tast-template.jinja2`? I think it's fine to hard-code it in this template since it's going to be the same for all Chrome OS tests.

The first strategy was adding context in template hard-coded, but..it will be a break in "kernel CI design" usually we define variables in test-configs. :)

Reply:
How about devices that also define a `context` template parameter though? I think they're going to collide and only one of them will actually be in the template.

For example:
```yaml
  bcm2837-rpi-3-b:
    mach: broadcom
    class: arm64-dtb
    boot_method: uboot
    context: {console_device: 'ttyS1', test_character_delay: 10}
    filters:
      - blocklist:
          defconfig: *device_config_filter
          kernel: ['v3.', '4.4-', '4.9-', 'v4.13']
```
That's why I thought, if the Chrome OS template updates the `context` dictionary then things will get merged together.

See the code in `kernelci.test`:
```python
def get_params(meta, target, plan_config, storage):
    # ...
    if rootfs:
        # ...
        params.update(rootfs.params)

    params.update(plan_config.params)
    params.update(target.params)


    return params
```
So we would need another PR to replace the plain `.update()` calls with a function to recursively update dictionaries inside each config params.

I think that can be done and it would seem like a good improvement, but maybe we can get this PR merged by hard-coding the parameters in the template and then improve it as a follow-up.